builders/build/customize/.pages

```diff
@@ -1,8 +1,9 @@
-title: Local Environment
+title: Customize Chain
 hide: false
 nav:
   - index.md
   - 'Prerequisites': 'prerequisites.md'
-  - 'Adding a Built-in Module': 'adding-built-in-pallet.md'
+  - 'Adding a Built-in Module': 'adding-built-in-module.md'
```

Comment:
@augustocollado was there a redirect created for this?

Reply:
Nope